fog/sample-paykit/src/cached_tx_data/mod.rs

```diff
@@ -133,7 +133,7 @@ impl CachedTxData {
             owned_tx_outs: Default::default(),
             key_image_data_completeness: BlockCount::MAX,
             latest_global_txo_count: 0,
-            latest_block_version: 1,
+            latest_block_version: 0,
```

Comment:
Might be better to use the `BlockVersion` constant here so it can be grepped.

Reply:
the problem with that is that, it may be impossible to convert the network's block version to a `BlockVersion` from our `mc-transaction-core`. I guess we could use `Option<BlockVersion>` but that felt messy

Reply:
Ah